Mathlib/Topology/Maps/Basic.lean

```diff
@@ -423,17 +423,30 @@ theorem preimage_frontier_eq_frontier_preimage (hf : IsOpenMap f) (hfc : Continu
 
 theorem of_isEmpty [h : IsEmpty X] (f : X → Y) : IsOpenMap f := of_nhds_le h.elim
 
+theorem clusterPt_comap (hf : IsOpenMap f) {x : X} {l : Filter Y} (h : ClusterPt (f x) l) :
+    ClusterPt x (comap f l) := by
+  rw [ClusterPt, ← map_neBot_iff, Filter.push_pull]
+  exact h.neBot.mono <| inf_le_inf_right _ <| hf.nhds_le _
+
 end IsOpenMap
 
 theorem isOpenMap_iff_nhds_le : IsOpenMap f ↔ ∀ x : X, 𝓝 (f x) ≤ (𝓝 x).map f :=
   ⟨fun hf => hf.nhds_le, IsOpenMap.of_nhds_le⟩
 
+theorem isOpenMap_iff_image_mem_nhds : IsOpenMap f ↔ ∀ x, ∀ s ∈ 𝓝 x, f '' s ∈ 𝓝 (f x) := by
+  simp only [isOpenMap_iff_nhds_le, le_map_iff]
```

Comment:
I'm not convinved by this lemma, vs just letting the caller rewrite by `le_map_iff` themselves.

Reply:
My only motivation for this lemma was that we have one of the implications but not the other.

Reply:
@eric-wieser Do you want me to delete this lemma before merging?

Reply:
I'll let you decide.